### PR TITLE
Improving docs for reddit question. 

### PR DESCRIPTION
**Pull request by mgwidmann**
https://www.reddit.com/r/elixir/comments/4t5d6n/question_in_a_form_what_does_the_as_atom_do/

### CODE REVIEW

File: lib/phoenix_html/form.ex [hunk 1]
```diff
@@ -205,8 +205,10 @@ defmodule Phoenix.HTML.Form do
 
   ## Options
 
-    * `:as` - the name to be used in the form. Automatically inflected
-      when a changeset is given
+    * `:as` - the server side parameter in which all params for this
+      form will be collected (i.e. `as: :user_params` would mean all fields
+      for this form will be accessed as `conn.params.user_params` server
+      side). May be inflected if a model is available.
```
Comment (josevalim) [inline lib/phoenix_html/form.ex]: s/model/changeset?

Comment (mgwidmann) [inline lib/phoenix_html/form.ex]: Ah sorry! I had an older version of the docs. 👍

**Comment by mgwidmann** [comment]
👍 Good to go now

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**